java/core/src/test/org/apache/orc/TestVectorOrcFile.java

```diff
@@ -88,32 +89,21 @@
 import java.util.Map;
 import java.util.Random;
 import java.util.TimeZone;
+import java.util.UUID;
 import java.util.function.IntFunction;
-
-import static junit.framework.TestCase.assertNotNull;
-import static org.junit.Assert.*;
+import java.util.stream.Stream;
```

Comment:
For `import` statement ordering, we need to introduce a rule to clean up this. I'll do that later.